README.md

```diff
@@ -3133,8 +3133,8 @@ Pass `-help` to VictoriaMetrics in order to see the list of supported command-li
      An optional list of labels to drop from samples before stream de-duplication and aggregation . See https://docs.victoriametrics.com/stream-aggregation.html#dropping-unneeded-labels
      Supports an array of values separated by comma or specified via multiple flags.
      Value can contain comma inside single-quoted or double-quoted string, {}, [] and () braces.
-  -streamAggr.ignoreOldSamples
-     Whether to ignore input samples with old timestamps outside the current aggregation interval. See https://docs.victoriametrics.com/stream-aggregation.html#ignoring-old-samples
+  -streamAggr.ignoreOldIntervals
+     Whether to ignore input intervals, that contain samples with old timestamps outside the current aggregation interval. See https://docs.victoriametrics.com/stream-aggregation.html#ignoring-old-intervals
```

Comment:
Shall we specify that this works on startup only and is designed to prevent producing discrepancies during backfilling?

Reply:
>Otherwise it won't protect us from spikes due network issues

My thinking was that it could do more harm than good. If during start-up we expect backfilling to happen, then during normal operation network issues are more like to be an anomaly.

Reply:
Why changing the name of the command-line flag is needed?